regulation-worker/internal/delete/kvstore/kvstore.go

```diff
@@ -28,11 +28,11 @@ func (kv *KVDeleteManager) Delete(ctx context.Context, job model.Job, destConfig
 	fileCleaningTime := stats.NewTaggedStat("file_cleaning_time", stats.TimerType, stats.Tags{"jobId": fmt.Sprintf("%d", job.ID), "workspaceId": job.WorkspaceID, "destType": "kvstore", "destName": destName})
 	fileCleaningTime.Start()
 	defer fileCleaningTime.End()
-	for _, user := range job.UserAttributes {
-		key := fmt.Sprintf("user:%s", user.UserID)
+	for _, user := range job.Users {
+		key := fmt.Sprintf("user:%s", user.ID)
 		err = kvm.DeleteKey(key)
```

Comment:
other attributes are not important for kvstore?

Reply:
while writing `kvstore` deleter we checked how it is stored it redis. And, we found that key there is `user:$USERID` . So, we are using it to delete a user data in case of `kvstore`.